File: app/src/main/java/embedded/block/vote/AdminVoteListAdapter.java
package embedded.block.vote;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.BaseAdapter;
import android.widget.TextView;

import java.util.ArrayList;

public class AdminVoteListAdapter extends BaseAdapter {
    private LayoutInflater inflater;
    private ArrayList<AdminVoteItem> data;
    private int layout;

    public AdminVoteListAdapter(Context context, int layout, ArrayList<AdminVoteItem> data){
        this.inflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
        this.data = data;
        this.layout = layout;
    }

    @Override
    public int getCount() {
        return data.size();
    }
    @Override
    public String getItem(int position) {
        return data.get(position).getVoteName();
    }
    @Override
    public long getItemId(int position) {
        return position;
    }
    @Override
    public View getView(int position, View convertView, ViewGroup parent) {
        if(convertView == null){
            convertView = inflater.inflate(layout,parent,false);
        }
        AdminVoteItem adminVoteItem = data.get(position);

        TextView voteNum = (TextView) convertView.findViewById(R.id.voteNum);
        voteNum.setText(Integer.toString(adminVoteItem.getVoteNum()));
        TextView voteName = (TextView) convertView.findViewById(R.id.voteName);
        voteName.setText(adminVoteItem.getVoteName());

        return convertView;
    }

}

File: app/src/main/java/embedded/block/vote/AdminVoteListAdapter.kt
package embedded.block.vote

import android.R
import android.content.Context
import android.text.Layout
import android.widget.TextView
import android.view.ViewGroup
import android.view.LayoutInflater
import android.view.View
import android.widget.BaseAdapter


class AdminVoteListAdapter(context: Context, val layout: Int, val data: ArrayList<AdminVoteItem>) : BaseAdapter() {
    val inflater: LayoutInflater
    init {
        this.inflater = LayoutInflater.from(context)
    }

    override fun getCount(): Int {
        return data.size
    }

    override fun getItem(position: Int): String {
        return data[position].voteName
    }

    override fun getItemId(position: Int): Long {
        return position.toLong()
    }

    override fun getView(position: Int, convertView: View?, parent: ViewGroup?): View {
        val view = convertView ?: inflater.inflate(layout, parent, false)

        view.findViewById<TextView>(embedded.block.vote.R.id.voteNum).text = data[position].voteNum.toString()
        view.findViewById<TextView>(embedded.block.vote.R.id.voteName).text = data[position].voteName

        return view
    }

}